src/interface/emacs/khoj.el

```diff
@@ -609,7 +609,7 @@ CONFIG is json obtained from Khoj config API."
   ;; POST provided config to khoj server
   (let ((url-request-method "POST")
         (url-request-extra-headers '(("Content-Type" . "application/json")))
-        (url-request-data (json-encode-alist config))
+        (url-request-data (encode-coding-string (json-encode-alist config) 'utf-8))
```

Comment:
All you need is to ingest a file whose name includes unicode character, for example 💀.org or something. Then you will see the error message. Since I already fixed on my side using this PR, I cannot provide the backtrace; I'll append to here if I found a time to reproduce it.

Reply:
Ah, interesting, didn't think of unicode filenames. Let me try repro the issue and fix on our end. Thanks for the explanation

Reply:
Thanks a ton for submitting a fix! I was able to repro the issue (and fix). Merging the change